src/devices_models/devices/hydro_generation.jl

```diff
@@ -220,10 +174,16 @@ function activepower_constraints!(
     parameters = model_has_parameters(psi_container)
     use_forecast_data = model_uses_forecasts(psi_container)
 
-    ts_data_active, constraint_data =
-        _get_time_series(psi_container, devices, model, x -> PSY.get_activepowerlimits(x))
-
     if !parameters && !use_forecast_data
```

Comment:
Is it possible for parameters to be true and use_forecast_data to be false?  This code doesn't handle that case.

Reply:
I don't think that case was ever handled.